tasks/request_files/request_files.py

```diff
@@ -36,6 +36,7 @@
 EVENT_INPUT_KEY = "input"
 
 CONFIG_ORCA_DEFAULT_BUCKET_OVERRIDE_KEY = "orcaDefaultBucketOverride"
+CONFIG_RESTORE_RETRIEVAL_TYPE_KEY = "orcaRestoreTypeOverride"
```

Comment:
Include "OVERRIDE" in key name.

Reply:
I concur. You may also want to include "DEFAULT" in the key name as posted in other comments.

Reply:
changed to `CONFIG_RESTORE_DEFAULT_RETRIEVAL_TYPE_OVERRIDE_KEY`